packages/mobile/src/store/sign-up/sagas/watchSignUpSucceededSaga.ts

```diff
@@ -0,0 +1,38 @@
+import { make } from 'common/store/analytics/actions'
+import { SIGN_UP_SUCCEEDED } from 'common/store/pages/signon/actions'
+import {
+  getEmailField,
+  getHandleField
+} from 'common/store/pages/signon/selectors'
+import { takeEvery, put, select } from 'typed-redux-saga'
+
+import { remindUserToTurnOnNotifications } from 'app/components/notification-reminder/NotificationReminder'
+import { EventNames } from 'app/types/analytics'
+
+export function* watchSignUpSucceeded() {
+  yield* takeEvery(SIGN_UP_SUCCEEDED, handleSignUpSucceeded)
+}
+
+function* handleSignUpSucceeded() {
+  const emailField = yield* select(getEmailField)
+  const handleField = yield* select(getHandleField)
+
+  // Record both CREATE_ACCOUNT_COMPLETE_CREATING and
+  // CREATE_ACCOUNT_FINISH events
+
+  yield put(
+    make(EventNames.CREATE_ACCOUNT_COMPLETE_CREATING, {
+      emailAddress: emailField.value,
+      handle: handleField.value
+    })
+  )
+
+  yield put(
+    make(EventNames.CREATE_ACCOUNT_FINISH, {
+      emailAddress: emailField.value,
+      handle: handleField.value
+    })
+  )
+
+  remindUserToTurnOnNotifications(put)
```

Comment:
do we want to yield call this? lol interesting we pass it put!

Reply:
I don't think we need to, just need to put this action in a best effort fashion. Nothing else is waiting on this saga

Yeah lol, it accepted dispatch from the component but `put` works the same!